test/app.js

```diff
@@ -0,0 +1,9 @@
+const tape = require('tape');
+const app = require('../src/app');
+
+
+tape('testing port', (t) => {
+  const expected = '3000';
+  t.equal(app.get('port'), expected, 'port should be set to 3000');
```

Comment:
Testing for the port? Even if I merge this, this should cause a test failure in any production because you can't control the production port. I'm amazed it escaped the travis testing, I'd say that this file is never run, and amazingly, I am correct.